src/Application.vala

```diff
@@ -66,6 +66,15 @@ public class Sideload.Application : Gtk.Application {
 
     }
 
+    public unowned string get_appstore_name () {
+        var appinfo = GLib.AppInfo.get_default_for_uri_scheme ("appstream://");
+        if (appinfo != null) {
+            return appinfo.get_name ();
+        } else {
+            return _("AppCenter");
```

Comment:
If AppCenter is covered by the AppInfo, should this just fall back to something more generic like, “your software center”?